### PR TITLE
StatsBar Component

### PR DESCRIPTION
**Pull request by skiran86**

### CODE REVIEW

File: src/components/statsBar/AllStatsController.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/statsBar/AllStatsController.js:1]: You don't need these controllers!! Can you just update the view? As we don't want to depend on redux at two places

File: src/components/statsBar/StatsView.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```
Comment (AjayDoddapaneni) [inline src/components/statsBar/StatsView.js:1]: This is the only file we need to publish. The rest can stay in respective projects

**Comment by AjayDoddapaneni** [comment]
@skiran86 Can you add a story component in this file with dummy data
https://github.com/CBIIT/bento-tools/blob/master/src/stories/index.js

File: src/components/statsBar/StatsView.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    margin: '8px 0px 0px -35px',
+  },
+});
+
+StatsBar.defaultProps = {
+  classes: {},
+  statsStyling: {},
+};
+
+const StyledStatsBar = withStyles(styles)(StatsBar);
```
Comment (AjayDoddapaneni) [inline src/components/statsBar/StatsView.js:111]: Can you add a comment TODO: need to change it to injectStyle sheet

**Review by AjayDoddapaneni — APPROVED** [review]
LGTM

File: src/components/statsBar/StatsView.js (hunk 3 above)
Comment (skiran86) [reply src/components/statsBar/StatsView.js:112]: Done

**Pull request merged**